lib/sequent/migrations/view_schema.rb

```diff
@@ -283,6 +282,27 @@ def migrate_offline
         raise e
       end
 
+      def migrate_dryrun(regex, group_exponent)
+        # Complete hack to override insert_ids
+        define_singleton_method(:insert_ids) do
```

Comment:
I would move adding these methods to the rake task so they can be "accidentally" be overwritten by calling this method in the real production environment.